pyproject.toml

```diff
@@ -94,10 +104,10 @@ target-version = "py310"
 
 [tool.ruff.per-file-ignores]
 "tests/state_test.py" = ["ANN001", "ANN201", "ANN401"]
-"beaker/lib/*.py" = ["N802"]  # lowercase function names
+"beaker/lib/*.py" = ["N802"]                           # lowercase function names
```

Comment:
that's a lot of blank spaces lol

Reply:
@achidlow weird, editorconfig linter acting out